aflorea/dev-test
# cache the values using Predis

This PR caches successful results for an hour.

---

**Testing**

- Install redis on a server
- Start the redis daemon
- run composer install to install predis
- change the IP for the redis server in index.php
Now the results are served from the redis instead of the endpoint.
Optionally, after a successful API call, the cache can viewed via redis-cli.

On the server where redis is running:
- run redis-cli
- get 3ev_cached_results
A serialized string should show containing information about the seasons/episodes.

---

**Deployment steps**

Merge branch `cache` into `master`

Install a redis server, change index.php to point to the right redis server.
